pkg/rtc/room.go

```diff
@@ -960,16 +964,26 @@ func (r *Room) pushAndDequeueUpdates(pi *livekit.ParticipantInfo, isImmediate bo
 	shouldSend := isImmediate || pi.IsPublisher
 
 	if existing != nil {
-		if pi.Sid != existing.Sid {
-			// session change, need to send immediately
-			isImmediate = true
-			existing.State = livekit.ParticipantInfo_DISCONNECTED
-			updates = append(updates, existing)
-		} else if pi.Version < existing.Version {
-			// out of order update
-			return nil
-		} else if shouldSend {
-			updates = append(updates, existing)
+		if pi.Sid == existing.Sid {
+			// same participant session
+			if pi.Version < existing.Version {
+				// out of order update
+				return nil
+			} else if shouldSend {
+				updates = append(updates, existing)
```

Comment:
This is how the older code was. Wondering if we should append this. Basically, `pi` (what this function is called with) is newer version and will be appended below in the check
```
    if shouldSend {
    }
```
So, this would place an older version first and the newer version of same participant later in the list. Can we drop this append of older version?

Reply:
Yeah I think we can drop the older version. I don't quite remember why I did this in the first place.

Reply:
Removing it in this [commit](https://github.com/livekit/livekit/pull/1372/commits/f880e02c2bb103b9d0a4cecc78143385569e0a6b).